solarforecastarbiter/reports/tests/test_reports_main.py

```diff
@@ -66,12 +67,43 @@ def test_full_render(mock_data, report_objects):
     report_md = main.render_raw_report(raw_report)
     body = template.report_md_to_html(report_md)
     full_report = template.full_html(body)
+    # at least one non whitespace character in body, usually caused
```

Comment:
I'm not sure I understand. Under some conditions, e.g. a pandoc version error, `template.full_html(body)` runs without raising an exception but produces an empty body tag?

Reply:
correct